Source/Events.Store.MongoDB/Streams/IEventsToStreamsWriter.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Dolittle.Runtime.Events.Store.Streams;
+using MongoDB.Driver;
+
+namespace Dolittle.Runtime.Events.Store.MongoDB.Streams
+{
+    /// <summary>
+    /// Defines a that can write events to a Stream.
+    /// </summary>
+    public interface IEventsToStreamsWriter
```

Comment:
The other interfaces are usually called `IWriteXToY` so we should probably follow the same pattern.
```suggestion
    public interface IWriteStoreEventsToStreams
```

Reply:
```suggestion
    public interface IWriteEventsToStreamCollection
```